api/repositories/droplet_repository_test.go

```diff
@@ -313,6 +313,88 @@ var _ = Describe("DropletRepository", func() {
 		})
 	})
 
+	Describe("GetDroplets", func() {
```

Comment:
If you agree to go with extending `ListDroplets` rather than introducing `GetDroplets`, then this describe would not be needed. Instead, `ListDroplets` tests should be extended to proove filtering by app guid